pytest_fixtures/component/os.py

```diff
@@ -17,16 +20,25 @@ def default_os(
     """
     os = getattr(request, 'param', None)
     if os is None:
-        search_string = 'name="RedHat" AND (major="6" OR major="7" OR major="8")'
+        default_name = settings.supportability.content_hosts.default_os_name
+        default_major_versions = " OR ".join(
+            [f'major="{v}"' for v in settings.supportability.content_hosts.rhel.versions]
+        )
+        search_string = f'name="{default_name}" AND ({default_major_versions})'
     else:
-        version = os.split(' ')[1].split('.')
-        search_string = f'family="Redhat" AND major="{version[0]}" AND minor="{version[1]}"'
-    os = entities.OperatingSystem().search(query={'search': search_string})[0].read()
+        name, version = os.split(' ')
+        version = version.split('.')
+        search_string = f'name="{name}" AND major="{version[0]}" AND minor="{version[1]}"'
+    os = (
+        session_target_sat.OperatingSystem()
```

Comment:
```suggestion
        session_target_sat.api.OperatingSystem()
```
Whoops I forgot you need the `.api` attribute